apps/examples/ble_rmc/ble_rmc_main.c

```diff
@@ -186,17 +186,41 @@ int ble_rmc_main(int argc, char *argv[])
 	RMC_LOG(RMC_TAG, "cmd : %s\n", argv[1]);
 
 	if (strncmp(argv[1], "init", 5) == 0) {
-		ret = ble_manager_init(&client_config, &server_config);
-		RMC_LOG(RMC_TAG, "init done[%d]\n", ret);
+		if (argc == 3 && strncmp(argv[2], "null", 5) == 0) {
+			ret = ble_manager_init(NULL);
+			if (ret != BLE_MANAGER_SUCCESS) {
+				if (ret != BLE_MANAGER_ALREADY_WORKING) {
+					RMC_LOG(RMC_TAG, "init with null fail[%d]\n", ret);
+					return 0;
+				}
+				RMC_LOG(RMC_TAG, "init is already done\n");
```

Comment:
How about changing the log as `init with null is already done.`?

Reply:
In this case, the user can not realize whether the config is set or not.

Reply:
Okay. I understand. Thanks.